src/Apps/Artwork/Components/ArtworkImageBrowser/ArtworkImageBrowser.tsx

```diff
@@ -89,8 +81,11 @@ export class LargeArtworkImageBrowser extends React.Component<
                       deepZoom={image.deepZoom}
                       enabled={image.is_zoomable}
                       isDefault={image.is_default}
+                      src={image.uri}
                     >
-                      <DesktopImage src={image.uri} width="100%" />
+                      <Flex justifyContent="center" alignItems="center">
+                        <DesktopImage alt={imageAlt} src={image.uri} />
```

Comment:
why is is `src` in `<Lightbox>` + `<DesktopImage>` needed? Can't one be derived from the other?

Reply:
`DesktopImage` is rendered only on the SSR pass and the `Lightbox` renders a separate `<img>`

Reply:
If you imagine `<Lightbox>` as a self contained thing, then piggybacking off of the image as is passed into `src` makes it configuration free, and all behavior can be contained within.

Inside of `<Lightbox>`, where it was mapping over children, the image can be rendered, with the container if need be. And then from an API perspective all that’s needed is

```js
<Lightbox src="path/to/image" ... />
```

which makes everything nice and portable. It also removes the need to duplicate `image.uri` in two separate places.

Reply:
That said, the currently-existing behavior as described [here](https://github.com/artsy/reaction/pull/2371#discussion_r281344381) is actually correct.

Reply:
Originally, `Lightbox` was just intended to be a HOC that could consume an Image component of some sort and decorate it with the things it needed to invoke a Lightbox.

From just reading over the PR this API seems a little deceptive to me. You're allowing children to be passed but *only* rendering the children when it's not SSR. This would be a really easy way for folks to unknowingly introduce a rehydration issue. 

I'm just worried it'd be confusing for folks. It should either always take a child (and render that appropriately based on context) or never take a child.

Reply:
👍 We were just chatting on a call and that's what we settled on; sticking to a simple `src` and ommitting children completely. That way it removes the duplication and the ambiguity.